src/pages/donate/[name].tsx

```diff
@@ -89,60 +56,22 @@ const DonateTo: NextPage = () => {
     }
   }, [transactionUri, streamer]);
 
-  if (isLoading) {
+  if (status === "error" && error instanceof Error) {
+    console.error({ error });
+    router.push("/donate");
```

Comment:
Don't navigate programmatically in the `render` method of component. 

See: https://reactrouter.com/docs/en/v6/components/navigate#navigate